tests/metrics.yaml

```diff
@@ -161,21 +161,21 @@ HostSystemPropertiesCollector:
 
 DatastoreStatsCollector:
   metrics:
-      - 'vrops_datastore_diskspace_freespace_gigabytes{datacenter="datacenter3",datastore="datastore1",hostsystem="hostsystem3",vccluster="cluster3",vcenter="vcenter1"}'
-      - 'vrops_datastore_diskspace_total_usage_gigabytes{datacenter="datacenter3",datastore="datastore2",hostsystem="hostsystem3",vccluster="cluster3",vcenter="vcenter1"}'
-      - 'vrops_datastore_diskspace_total_usage_gigabytes{datacenter="datacenter3",datastore="datastore1",hostsystem="hostsystem3",vccluster="cluster3",vcenter="vcenter1"}'
-      - 'vrops_datastore_diskspace_freespace_gigabytes{datacenter="datacenter3",datastore="datastore2",hostsystem="hostsystem3",vccluster="cluster3",vcenter="vcenter1"}'
-      - 'vrops_datastore_diskspace_freespace_gigabytes{datacenter="datacenter3",datastore="datastore3",hostsystem="hostsystem3",vccluster="cluster3",vcenter="vcenter1"}'
-      - 'vrops_datastore_diskspace_capacity_gigabytes{datacenter="datacenter3",datastore="datastore1",hostsystem="hostsystem3",vccluster="cluster3",vcenter="vcenter1"}'
-      - 'vrops_datastore_diskspace_capacity_gigabytes{datacenter="datacenter3",datastore="datastore3",hostsystem="hostsystem3",vccluster="cluster3",vcenter="vcenter1"}'
-      - 'vrops_datastore_diskspace_capacity_gigabytes{datacenter="datacenter3",datastore="datastore2",hostsystem="hostsystem3",vccluster="cluster3",vcenter="vcenter1"}'
-      - 'vrops_datastore_diskspace_total_usage_gigabytes{datacenter="datacenter3",datastore="datastore3",hostsystem="hostsystem3",vccluster="cluster3",vcenter="vcenter1"}'
+      - 'vrops_datastore_diskspace_freespace_gigabytes{datacenter="datacenter3",datastore="datastore1",hostsystem="hostsystem3",type="None",vccluster="cluster3",vcenter="vcenter1"}'
```

Comment:
I would like to test something else but `None` here. 
Ideally we would test all these conditions
```
            return "vmfs_p_ssd"
            return "vmfs_s_hdd"
            return "ephemeral"
            return "Management"
            return "vVOL"
            return "local"```